tests/unittest_inference.py

```diff
@@ -4605,6 +4605,20 @@ class LenInvalid(object):
             inferred = next(node.infer())
             self.assertEqual(inferred, util.Uninferable)
 
+    def test_class_subscript(self) -> None:
+        node = extract_node(
+            """
+        class Foo:
+            def __class_getitem__(cls, *args, **kwargs):
+                return cls
+
+        Foo[True]
```

Comment:
```suggestion
        Foo[int]
```
This shouldn't change much, but usually the subscript is a type.